c-cannons/bangazon-terminal-interface
# Ep sprint 1 test

### Pull Requests

#### Description
 * added Customer Controller
* added Customer Model
* added tests for Customer Model
* Set up Main Menu Functionality

#### Steps to Test
* run `bundle exec rspec` from root folder
* make sure database in the customer model test is linked

#### Link to Feature Ticket

#1  #2
